events/wc_gnoll_events.txt

```diff
@@ -0,0 +1,86 @@
+#Skinning Festival
+
+namespace=WCGW
+
+#Liege sends letters and prepares
+character_event={
+	id=WCGW.100
+	desc=WCGWDESC.100
+	is_triggered_only=yes
+
+	option={
+		name=EVTOPTA_WCGW.100
+		wealth = -25
+		set_character_flag = holding_skinning_festival
+		set_character_flag = recent_minor_decision
+		set_character_flag = do_not_disturb
+		hidden_tooltip = { character_event = { id = WCGW.699 days = 150 } } # Safety catch flag clearing
+		any_vassal={
+			limit={
+				prisoner=no
+				age=16
+				culture_group=gnoll_group
+				war = no
+				NOT = { trait = incapable }
+				liege=ROOT
+			}
+			character_event = { id = WCGW.102 tooltip = EVTTOOLTIP_WCGW_102 days = 2 }
+		}
+		hidden_tooltip = { character_event = { id = WCGW.101 days = 10 } }
+		if = {
+			limit = {
+				has_dlc = "Reapers"
+			}
+			custom_tooltip = {
+				text = capital_prospers_custom_tooltip
+				hidden_tooltip = {
+					capital_scope = {
+						change_variable = { which = prosperity_value value = 8 }
+					}
+				}
+			}
+		}
+	}
+	option={
+		name=EVTOPTB_WCGW.100
+		wealth = 25
+		clr_character_flag = holding_skinning_festival
+		clr_character_flag = recent_minor_decision
+		clr_character_flag = do_not_disturb
+	}
+}
+#Liege skinning begins
+character_event={
+	id=WCGW.101
+	desc=WCGWDESC.101
+	is_triggered_only=yes
+
+	option={
+		name=EVTOPTA_WCGW.101
+		random_courtier = {
+			limit = {
+				prisoner = yes
+				host = { character = ROOT }
+				can_be_sacrificed_trigger = yes
+			}
+			hidden_tooltip = { character_event = { id = WCGW.103 } }
+		}
+	}
+}
+#Vassal gets letter
+character_event={
+	id=WCGW.102
+	desc=WCGWDESC.102
+	is_triggered_only=yes
+	border=GFX_event_letter_frame_diplomacy
+
+	option={
+		name=EVTOPTA_WCGW.102
+	}
+	option={
+		name=EVTOPTB_WCGW.102
+		effect={
+			gnollish_opinion = -10
+		}
```

Comment:
Wrong syntax. Most likely you need something like...
```suggestion
		add_character_modifier = { name = declined_skinning_festival years = 2 }
```
Check `add_character_modifier` [here](https://ck2.paradoxwikis.com/Commands).
Feel free to change the duration: `years = 2`.